src/middlewares/auth.js

```diff
@@ -67,7 +67,10 @@ class Authentication {
    */
   static decodeJwt(token) {
     let payload = null;
-    payload = jwt.decode(token, index.secret);
+    payload = jwt.verify(token, index.secret, (err, decoded) =>{
```

Comment:
is there a reason we are changing the existing line to this?

Reply:
Yeah we had issues using decode, so verify provided the solution